templates/Makefile.j2

```diff
@@ -118,11 +118,11 @@ tuning_reports/{{ wiki.name }}.{{ model_name }}.md: \
 		--cv-timeout 60 \
 		--debug > $@
 
-models/{{ wiki.name }}.{{ model_name }}.gradient_boosting.model: \
+models/{{ wiki.name }}.{{ model_name }}.{% if model.rf %}rf{% else %}gradient_boosting{% endif %}.model: \
```

Comment:
IMO this is too much logic for a template.  Not sure what the best alternative is, though.  Normally I would suggest writing template helpers, but this would be our first.  What do you think?

Reply:
I would love to see some docs, let's make a phab card and do it soon.